build-farm/platform-specific-configurations/windows.sh

```diff
@@ -27,21 +27,27 @@ TOOLCHAIN_VERSION=""
 
 # Any version above 8 (11 for now due to openjdk-build#1409
 if [ "$JAVA_FEATURE_VERSION" -gt 11 ]; then
-    BOOT_JDK_VERSION="$((JAVA_FEATURE_VERSION-1))"
+    if [ "$ARCHITECTURE" == "aarch64" ]; then
+      # Windows aarch64 cross compiles requires same version boot jdk
+      BOOT_JDK_VERSION="$((JAVA_FEATURE_VERSION))"
+    else
+      BOOT_JDK_VERSION="$((JAVA_FEATURE_VERSION-1))"
+    fi
     BOOT_JDK_VARIABLE="JDK$(echo $BOOT_JDK_VERSION)_BOOT_DIR"
     if [ ! -d "$(eval echo "\$$BOOT_JDK_VARIABLE")" ]; then
       bootDir="$PWD/jdk-$BOOT_JDK_VERSION"
       # Note we export $BOOT_JDK_VARIABLE (i.e. JDKXX_BOOT_DIR) here
       # instead of BOOT_JDK_VARIABLE (no '$').
       export ${BOOT_JDK_VARIABLE}="$bootDir"
       if [ ! -d "$bootDir/bin" ]; then
-        echo "Downloading GA release of boot JDK version ${BOOT_JDK_VERSION}..."
-        releaseType="ga"
         # This is needed to convert x86-32 to x32 which is what the API uses
         case "$ARCHITECTURE" in
           "x86-32") downloadArch="x32";;
+          "aarch64") downloadArch="x64";;
```

Comment:
I'm not convinced that's right once we have aarch64 released for that version +1